cli/src/cli-cmd.c

```diff
@@ -379,14 +379,12 @@ cli_cmd_submit(struct rpc_clnt *rpc, void *req, call_frame_t *frame,
 }
 
 int
-cli_cmd_pattern_cmp(void *a, void *b)
+cli_cmd_pattern_cmp(const void *a, const void *b)
 {
-    struct cli_cmd *ia = NULL;
-    struct cli_cmd *ib = NULL;
+    struct cli_cmd *ia = (struct cli_cmd *)a;
+    struct cli_cmd *ib = (struct cli_cmd *)b;
```

Comment:
We should also use `const` here. Otherwise the `const` in the arguments is useless.

```suggestion
    const struct cli_cmd *ia = (const struct cli_cmd *)a;
    const struct cli_cmd *ib = (const struct cli_cmd *)b;
```

Reply:
OK